src/citrine/resources/table_config.py

```diff
@@ -156,7 +156,11 @@ def add_columns(self, *,
             datasets=copy(self.datasets),
             rows=copy(self.rows),
             variables=copy(self.variables) + [variable],
-            columns=copy(self.columns) + columns
+            columns=copy(self.columns) + columns,
+            config_uid=copy(self.config_uid),
+            version_uid=copy(self.version_uid),
```

Comment:
I don't think we want to copy the version_uid, since that is provided by the service when the config is registered.
```suggestion
```